content/en/metrics/open_telemetry/_index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Send Metrics from OpenTelemetry to Datadog
+kind: documentation
+further_reading:
+    - link: 'tracing/trace_collection/open_standards/'
+      tag: 'Documentation'
+      text: 'Learn more about OpenTelemetry'
+---
+
+## Overview
+
+[OpenTelemetry][1] (OTel) is an open source observability framework that provides IT teams with standardized protocols and tools for collecting and routing telemetry data. Created as an incubator project by the [Cloud Native Computing Foundation][2] (CNCF), OTel provides a consistent format for instrumenting, generating, gathering, and exporting application telemetry data—namely metrics, logs, and traces—to monitoring platforms for analysis and insight.
+
+## Setup
+
+To send OTel metrics to Datadog, you have two options: the Datadog Agent, or the OTel Collector. Using the Datadog Agent enables you to keep using all [Agent functionalities][3]. For a more vendor-agnostic setup, use the OTel Collector.
+
+### Datadog Agent setup
+
+**OTel SDKs/Libraries** -> **Datadog Agent** -> **Datadog**
+
+#### Configuration
+
+1. [Instrument your system with OpenTelemetry][4].
+2. [Enable OTLP ingestion][5] on the Datadog Agent.
+3. [Send OTLP telemetry][6] from your system to Datadog.
+
+### OTel Collector setup
+
+**OTel SDKs/Libraries** and other sources -> **OTel Collector** with the Datadog Exporter -> **Datadog**
+
+Using this setup facilitates collecting telemetry data from sources besides OTel SDKs (for example, other libraries, Prometheus, etc.), as well as routing data to multiple vendors. 
+
+#### Configuration
+
+1. [Instrument your system with OpenTelemetry][4].
+2. Follow the [OTel Collector documentation][7] to install the `opentelemetry-collector-contrib` distribution, or any other distribution that includes the Datadog Exporter.
+3. Add a `datadog` exporter to your [OTel configuration YAML file][8]. The following is a minimum configuration file to retrieve system metrics:
+   ```yaml
+   receivers:
+     hostmetrics:
+       scrapers:
+         load:
+         cpu:
+         disk:
+         filesystem:
+         memory:
+         network:
+         paging:
+         process:
+
+   processors:
+     batch:
+       timeout: 10s
+
+   exporters:
+     datadog:
+       api:
+         key: "<API key>"
+         site: "<Datadog site>"
+        
+   service:
+     pipelines:
+       metrics:
+         receivers: [hostmetrics]
+         processors: [batch]
+         exporters: [datadog]
+   ```
+   Add your [Datadog API key][9] and [site][10] (defaults to `datadoghq.com`).
+   See the [sample configuration file][11] for other available options.
```

Comment:
Please update the link to this one: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/datadogexporter/examples 
This folder include sample configuration file for both kubernetes and standard host config file templates.